src/components/LicenseRevokeModal/index.jsx

```diff
@@ -123,12 +122,6 @@ class LicenseRevokeModal extends React.Component {
               assign <strong>{user.userEmail}</strong> to another license, but they
               will need to re-enroll in any course after being assigned a new license.
             </p>
-            {this.shouldRenderRevocationCapMessaging() && licenseOverview.assigned > 0 && (
-              <p>
-                You have <strong>{licenseOverview.assigned}</strong> licenses that are
-                in pending status that could be removed or re-assigned.
-              </p>
-            )}
```

Comment:
After a conversation with UX, we are opting to remove this sentence for now rather than investing in refactoring to make the `licenseOverview.assigned` count used here not consider an active search query. If UX decides its crucial, we'll come back to it.